src/Kafka/ConsumerGroups/components/DeleteConsumerGroup.tsx

```diff
@@ -0,0 +1,62 @@
+import { Alert, Text } from "@patternfly/react-core";
+import type { FunctionComponent } from "react";
+import { useTranslation } from "react-i18next";
+import { DeleteModal } from "../../../shared";
+import type { ConsumerGroupState } from "../types";
+
+export type DeleteConsumerGroupProps = {
+  isModalOpen: boolean;
+  onClose: () => void;
+  onDeleteConsumer: () => void;
+  state: ConsumerGroupState;
+  consumerName: string;
+};
+
+export const DeleteConsumerGroup: FunctionComponent<
+  DeleteConsumerGroupProps
+> = ({ isModalOpen, onClose, onDeleteConsumer, state, consumerName }) => {
+  const { t } = useTranslation("kafka");
+
+  const isConsumerConnected = state === "Stable";
+
+  return (
+    <div>
+      <DeleteModal
+        isModalOpen={isModalOpen}
+        title={t("consumerGroup.delete")}
+        isDeleting={isConsumerConnected}
+        onCancel={onClose}
+        onDelete={onDeleteConsumer}
+        variant={"non-destructive"}
+        appendTo={() => {
+          return document.getElementById("root") || document.body;
+        }}
+      >
+        {!isConsumerConnected && (
+          <Text id="modal-message">
+            <label
+              htmlFor="instance-name-input"
+              dangerouslySetInnerHTML={{
+                __html: t("common:confirm_delete_modal_text", {
+                  name: consumerName,
+                }),
+              }}
+            />
```

Comment:
```suggestion
            <label htmlFor="instance-name-input">
              {t("common:confirm_delete_modal_text", {
                name: consumerName
               })}
            </label>
```

Would this get us the same result?